boefjes/boefjes/plugins/kat_testssl_sh_ciphers/main.py

```diff
@@ -26,13 +27,14 @@ def run(boefje_meta: BoefjeMeta) -> list[tuple[set, bytes | str]]:
         detach=True,
     )
 
-    container.wait()
-
-    output = get_file_from_container(container, "tmp/output.json")
-
     try:
+        container.wait(timeout=300)
+        output = get_file_from_container(container, "tmp/output.json")
+    except (docker.errors.DockerException, RequestException) as e:
```

Comment:
It's better to stick to the [API's expected exceptions](https://docker-py.readthedocs.io/en/stable/containers.html#docker.models.containers.Container.wait): `requests.exceptions.ReadTimeout` and `docker.errors.APIError`.

Reply:
Yeah its weird, because I have tried those and it doesnt catch them correctly... Will try again tomorrow

Reply:
For this, you need to upgrade the `docker` package to the latest version (7.0.0)

Reply:
We use docker V6, not sure if we should upgrade for the sake of this PR